openedx/adg/lms/webinars/tests/test_tasks.py

```diff
@@ -0,0 +1,31 @@
+"""
+All tests for webinar tasks
+"""
+from datetime import datetime, timedelta
+
+import pytest
+
+from openedx.adg.lms.webinars.tasks import task_reschedule_webinar_reminders
+
+from .factories import WebinarFactory, WebinarRegistrationFactory
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'new_start_time, expected_call_count', [
+        (datetime.now() + timedelta(days=1), 1),
+        (datetime.now() + timedelta(days=8), 2),
+    ]
+)
+def test_task_send_mandrill_email_successfully(new_start_time, expected_call_count, mocker):
+    """
+    Tests if reschedule_webinar_reminders is called properly when the start_time is updated.
+    """
+    mock_reschedule = mocker.patch('openedx.adg.lms.webinars.tasks.reschedule_webinar_reminders')
+
+    webinar = WebinarFactory()
```

Comment:
There is already a fixture, by the name 'webinar', created for this purpose in `conftest.py`, hence it will be accessible here as well.